gauntlet/packages/gauntlet-solana-contracts/networks/.env.local

```diff
@@ -1,10 +1,10 @@
 NODE_URL=http://127.0.0.1:8899
-PROGRAM_ID_OCR2=E3j24rx12SyVsG6quKuZPbQqZPkhAUCh8Uek4XrKYD2x
-PROGRAM_ID_ACCESS_CONTROLLER=2ckhep7Mvy1dExenBqpcdevhRu7CLuuctMcx7G9mWEvo
-PROGRAM_ID_STORE=9kRNTZmoZSiTBuXC62dzK9E7gC7huYgcmRRhYv3i4osC
+PROGRAM_ID_OCR2=cjg3oHmg9uuPsP8D6g29NWvhySJkdYdAo9D25PRbKXJ
```

Comment:
Local was not updated to latest

Reply:
i'm not sure i understand why the local IDs are being updated? this change would break the local testing environment, and the keys already match the ones stored under `contracts/artifacts/localnet`

Reply:
True you're right, I was taking the ones in the `declare_id`s, let me revert it